src/stylesheets/layouts/_page.scss

```diff
@@ -0,0 +1,44 @@
+sds-page {
```

Comment:
most of these styles are border related css - I think it will make sense to create a borders file. After moving the border styles from this file. The rest of the styles could be achieved by utilities.

Reply:
Moved border and sizing out to their own files